Sprint-1/explore/objects.md

```diff
@@ -5,12 +5,18 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
 What output do you get?
+I get the following output: ƒ log() { [native code] }
 
 Now enter just `console` in the Console, what output do you get back?
+I get the following output: console {debug: ƒ, error: ƒ, info: ƒ, log: ƒ, warn: ƒ, …}
 
 Try also entering `typeof console`
+I get the following output: 'object'
 
 Answer the following questions:
 
 What does `console` store?
+The console effectively prints information for diagnostic purposes.
```

Comment:
That's not what the question asked. What you described are the "things" stored inside `console`. I wonder what ChatGPT would say.